Apps/Admin/Client/Store/Analytics/AnalyticsBaseAction.cs

```diff
@@ -0,0 +1,45 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.Admin.Client.Store.Analytics;
+
+using System;
+
+/// <summary>
+/// The base class for a csv export load action.
```

Comment:
Nit: "a csv export load action" can be replaced with "an analytics load action" to make it more of a general description.